packages/zk-certificates/scripts/decryptZkCert.ts

```diff
@@ -0,0 +1,41 @@
+/* Copyright (C) 2024 Galactica Network. This file is part of zkKYC. zkKYC is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. zkKYC is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>. */
+
+import { decryptSafely, getEncryptionPublicKey } from '@metamask/eth-sig-util';
+
+import zkCert from '../../../test/encryptedZkCert.json';
+import { testHolder } from '../../snap/test/constants.mock';
+// This is the zkCert you want to decrypt
```

Comment:
Due to auto-formatting of the imports, this comment is probably not where you wanted it to be. Let's remove it, because the next big comment is descriptive enough

Reply:
Thx, removed it.